generated/niswitch/niswitch.proto

```diff
@@ -7,7 +7,7 @@
 syntax = "proto3";
 
 option java_multiple_files = true;
-option java_package = "com.ni.grpc.niswitch";
+option java_package = "com.ni.grpc.switch";
```

Comment:
Cool, yeah we talked about this and I think it's fine to update the name here to be more in line with the other .proto files.